CHANGELOG.md

```diff
@@ -3,7 +3,11 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/) and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## v0.10.0 - not released yet
+## Unreleased
```

Comment:
Could you revert this line to mentioning v0.10.0 as it's the version in the `Cargo.toml` on the github tip.